Gems/ROS2/Code/Source/RobotImporter/URDF/JointsMaker.cpp

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "RobotImporter/URDF/JointsMaker.h"
+#include "RobotImporter/URDF/PrefabMakerUtils.h"
+#include "RobotImporter/Utils/TypeConversions.h"
```

Comment:
Include adjustments (both angle brackets for the RobotImporter/Utils directory and the removal of the RobotImported/URDF directory:
```suggestion
#include "JointsMaker.h"
#include "PrefabMakerUtils.h"
#include <RobotImporter/Utils/TypeConversions.h>
```